### PR TITLE
Fixes for the benat tests

### PR DESCRIPTION
**Pull request by dantleech**

### CODE REVIEW

File: src/Sulu/Bundle/ContentBundle/Tests/Features/content_type_url.feature [hunk 1]
```diff
@@ -6,7 +6,7 @@ Feature: Url content type
     Background:
         Given there exists a page template "url_page" with the following property configuration
         """
-        <property name="url" type="url">
+        <property name="this_url" type="url">
```
Comment (dantleech) [inline src/Sulu/Bundle/ContentBundle/Tests/Features/content_type_url.feature]: Hmm. This was strange.

Basically, the system doesn't seem to care if there are two properties with the same name, both here and in `develop`. The difference is that now, the second will replace the first, and the second does not have the `sulu.rlp` flag.

File: src/Sulu/Bundle/ContentBundle/Search/EventListener/HitListener.php [hunk 2]
```diff
@@ -36,6 +36,7 @@ public function __construct(RequestAnalyzerInterface $requestAnalyzer)
      */
     public function onHit(HitEvent $event)
     {
+        die('asd');
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Search/EventListener/HitListener.php]: That looks wrong :-)

Comment (dantleech) [inline src/Sulu/Bundle/ContentBundle/Search/EventListener/HitListener.php]: hmm.

**Comment by dantleech** [comment]
Updated.

**Comment by danrot** [comment]
Waiting for tests.

**Pull request merged**